.travis.yml

```diff
@@ -5,12 +5,13 @@ addons:
   postgresql: 9.6
 script:
 - bundle exec rails db:{create,migrate} RAILS_ENV=test
+- RAILS_ENV=test bundle exec rails webpacker:compile
```

Comment:
Adding this line DID change the travis logs, which now state that `Webpacker requires Node.js ">=10.17.0" and you are using v8.12.0` So perhaps it is trying to now run Webpacker, but cannot succeed